hazelcast/src/main/java/com/hazelcast/map/impl/nearcache/NearCacheImpl.java

```diff
@@ -84,12 +86,13 @@ public NearCacheImpl(String mapName, NodeEngine nodeEngine, SizeEstimator<NearCa
 
         Config config = nodeEngine.getConfig();
         NearCacheConfig nearCacheConfig = config.findMapConfig(mapName).getNearCacheConfig();
+        EvictionConfig evictionConfig = initDefaultMaxSize(nearCacheConfig.getEvictionConfig());
```

Comment:
This initializes the old default max size value, if no size was configured by the user.